django_grpc_framework/test.py

```diff
@@ -111,6 +111,11 @@ def add_generic_rpc_handlers(self, generic_rpc_handlers):
         _validate_generic_rpc_handlers(generic_rpc_handlers)
         self.rpc_method_handlers.update(generic_rpc_handlers[0]._method_handlers)
 
+    def add_registered_method_handlers(self, service_name, method_handlers):
```

Comment:
Do I understand correctly that if we use `1.64.1` or newer we don't need this?

Reply:
I think we wouldn't need this if `FakeServer` was inheriting from `Server` class defined in `grpcio`.

The newest version of `grpcio` (`1.65.2`) adds a line `server.add_registered_method_handlers(...)` to the code generated from proto files, where in our case `server` is an instance of `FakeServer`, so the code simply crashes.

Reply:
nit: newest is `v1.65.4`

Reply:
They are moving fast, it was the newest on Friday when I updated it :smile: